src/index.js

```diff
@@ -131,8 +131,8 @@ class Service {
 
     let { query, options } = multiOptions(id, params, this.id);
 
-    // We can not update the id
-    delete data[this.id];
```

Comment:
Not being able to change `_id` is a database restriction we can do nothing about. I would say we should allow changing any other id because it is much easier to add a hook with `delete hook.data.id` if you want to prevent it than not having the option at all.